docs/docsite/rst/user_guide/intro_inventory.rst

```diff
@@ -94,8 +94,8 @@ In the above example, trying to ansible against the host alias "jumper" (which m
 Note that this is using a feature of the inventory file to define some special variables.
 Generally speaking, this is not the best way to define variables that describe your system policy, but we'll share suggestions on doing this later.
 
-.. note:: Values passed in the INI format using the ``key=value`` syntax are not interpreted as Python literal structure
-          (strings, numbers, tuples, lists, dicts, booleans, None), but as a string. For example ``var=FALSE`` would create a string equal to 'FALSE'.
+.. note:: Values passed in the INI format using the ``key=value`` syntax are interpreted as Python literal structure
```

Comment:
this is incorrect, the previous text was correct, they are 'NOT" interpreted as python literals

Reply:
actually that is 1/2 correct ... i'll create PR that explains the nuance here cause this is dependent on where the variable is declared.